SampleOfferWithoutReward.sol

```diff
@@ -101,6 +136,14 @@ contract SampleOfferWithoutReward {
             paidOut += amount;
     }
 
+    function withdraw() {
+        var daysSinceSignoff = now - dateOfSignature; // TODO: Can be negative?
```

Comment:
`daysSinceSignOff` needs a division by 1 days here, since now it's seconds since sign off